README.md

```diff
@@ -181,6 +181,12 @@ Use the following command to generate OpenAPI specification:
 bundle exec rake rswag:specs:swaggerize
 ```
 
+We use [Tech Docs](https://github.com/alphagov/tech-docs-gem) to build documentation. To update documentation, the relevant files can be bound in `/docs`. The following command will then build the static site
+
+```sh
+cd docs && bundle install && bundle exec middleman build --build-dir=../public
```

Comment:
I have a feeling we should add the generated files to `.gitignore`, otherwise they'll end up getting checked in at some point.

Reply:
we won't need to do this any more as it's done by the docker build. I will update the docs so people use middleman directly

Reply:
docs have now been updated so we shouldn't need to do the gitignore